src/weight/randomWeight.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  calculateWeight: (profile) => {
```

Comment:
What's this for?

Reply:
That was more meant as a test of the system, so it's not really needed.
_But it's personally the best system there is. /s_